receiver/googlecloudspannerreceiver/internal/statsreader/projectreader_test.go

```diff
@@ -20,35 +20,27 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
 	"github.com/stretchr/testify/require"
 	"go.uber.org/zap/zaptest"
 
 	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/googlecloudspannerreceiver/internal/metadata"
 )
 
 type mockCompositeReader struct {
```

Comment:
Is this mock following any interface? if yes worth declaring it here.

Reply:
It is declared in main source code, not in tests.